apps/patient/src/api/internal.ts

```diff
@@ -34,29 +49,48 @@ export const getPharmacies = async ({
   isOpenNow: boolean;
   is24hr: boolean;
   name?: string;
+  includePrice?: boolean;
 }) => {
   try {
     const now = new Date();
-    const response = await graphQLClient.GetPharmaciesByLocation({
-      location: {
-        radius: 100,
-        ...searchParams
-      },
-      limit,
-      offset,
-      openAt: isOpenNow ? now : undefined,
-      is24hr,
-      name
+
+    // either the pharmaciesByLocation or pharmaciesWithPriceByLocation query is used
+    // depending on the includePrice flag
+    const fn = includePrice
+      ? graphQLClient.GetPharmaciesWithPriceByLocation
+      : graphQLClient.GetPharmaciesByLocation;
+    return formatPharmacies(
+      await fn({
+        location: {
+          radius: 100,
+          ...searchParams
+        },
+        openAt: isOpenNow ? now : undefined,
+        is24hr,
+        ...(!includePrice ? { limit, offset, name } : {})
```

Comment:
Why dont we have limit/offset here?

Reply:
I didn't see it in on the services query, @jknapp25 is that something we plan on adding before releasing this?

Reply:
it got added on getPharmacyPrices...

Reply:
So whats the deal here?